Docker/adaguc-services-config.xml

```diff
@@ -16,6 +16,7 @@
     <export>ADAGUC_LOGFILE=/var/log/adaguc/adaguc-server.log</export>
     <export>ADAGUC_FONT=/adaguc/adaguc-server-master/data/fonts/FreeSans.ttf</export>
     <export>ADAGUC_ONLINERESOURCE={ENV.EXTERNALADDRESS}/adaguc-services/adagucserver?</export>
+    <export>ADAGUC_CHECKER_DIR={ENV.ADAGUC_CHECKER_DIR}</export>
```

Comment:
Can be removed, not used anywhere.

Reply:
Done. Commit e7df138088b790ecef9defa310741b7615288120